docs/OTAnalytics/usage-ui.md

```diff
@@ -0,0 +1,434 @@
+3. **Sections/Flows**: Create, edit, and delete sections and flows. The flow names are shown with numbers indicating the number of assigned tracks per flow.
+4. **Analysis**: Start the export of analyzed data.
+
+The **Workspace** displays the tracks and video frames.
+It is also where the geometry of sections can be created or edited.
+Below the workspace, the **Track Statistics** section provides key statistics about the displayed tracks.
+
+Tracks can be customized using different **Visualization Layers**, while the number of displayed tracks can be adjusted through the **Visualization Filters** located below the workspace.
+
+![User Interface of OTAnalytics](usage-ui/user-interface.png)
+
+## Project Setup
+
+The project must first be named.
+The project name is entered in the corresponding Name field.
+The start time (date and time) of the first video must then be entered in the corresponding Start date fields.
+It is possible to enter in ISO 8601 format (YYYY-MM-DD) or German date format (DD.MM.YYYY).
+
+!!! info
+    OTAnalytics assumes that all selected videos are contiguous in time.
+    If the videos to be processed have a time gap (e.g., videos from three days from 6:00 a.m. to 10:00 p.m.), several projects with chronologically contiguous videos must be created.
+
+    It is recommended to choose a unique name (e.g. name of the measuring point, camera number; or a combination of several unique details).
+
+To analyze tracks in OTAnalytics, you must load the track files (.ottrk).
+This can be done using the *Add tracks...* button, which opens a file browser to select the respective files.
+Multiple track files can be added at once.
+If a corresponding video file exists, OTAnalytics will automatically load it alongside the track file.
+If no associated video is found, the track file cannot be loaded.
+
+Alternatively, you can add videos only.
+This is particularly useful for configuring an analysis handled by the platomo team.
+To add videos, use the *Add...* button, and to remove them from the configuration, use the *Remove* button.
+
+!!! tip
+    Added videos are displayed in alphabetical order in the “Videos” overview panel and are later processed in this order.
+    The videos should therefore be named in such a way that the chronological order matches the alphabetical order.
+    Additional zero-padding (e.g., `01`, `02`) is helpful if you want to ensure correct alphabetical sorting.
+
+When you click a video in the configuration bar, the first image of the video is displayed as the background image.
+This allows you to check whether all videos have been selected correctly.
+
+### Save to configuration file (otconfig file)
+
+Once all the required videos have been added and all the flows have been created, the project is saved.
+
+1. Click on the *Save as...* button in the Project section of the configuration bar.
+1. Select a suitable filename in the file browser and save it in the same folder as the videos. 
+
```

Comment:
@briemla: Please correct and, if necessary, differeniate between otflow and otconfig